examples/connectivity/plot_adhd_covariance.py

```diff
@@ -5,8 +5,8 @@
 This example shows how to extract signals from regions defined by an atlas,
 and to estimate a covariance matrix based on these signals.
 """
- 
-plotted_subject = 0  # subject to plot
+n_subjects = 10  # # subjects to consider for group-sparse covariance (max: 40)
```

Comment:
you have two `#` now. Maybe it's just me but I would remove the (max: 40): this information is mentioned in fetch_adhd docstring.